tools/buildx-gen.sh

```diff
@@ -58,14 +58,17 @@ for file in "$@"; do
     cat <<EOF >> "${config}"
 target "$image-$variant" {
     context = "${out}/${file}"
+    
     dockerfile = "Dockerfile.$image"
-    tags = ["${HUB}/${image}:${tag}"]
+    tags = ["${HUB}/${image}:${tag}-${TARGET_ARCH}"]
```

Comment:
I don't think we want to change the image names for the standard arch, maybe just append it if a notstandard?

Reply:
we could change it by `tag=${version}-${arch}`. i will revert it.

cause the istio/build-tools images not support oci.
so multi stage docker build is not possible now, we have to build multi tag.